packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -117,6 +117,18 @@ export class Loan {
     return await this.client.call('listcollateraltokens', [], 'bignumber')
   }
 
+  /**
+   * Get collateral token.
+   *
+   * @param {GetCollateralToken} [collateralToken = {}]
+   * @param {string} [collateralToken.token] Symbol or id of collateral token
```

Comment:
Does "id" work?

Reply:
The comment text is copied from Ain. Id does not work here. I have changed the text to `Symbol of collateral token`